[2025-03-06T06:18:10.815+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-06T06:18:10.819+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_horoscope_dag.send_email scheduled__2025-03-05T06:17:53.686903+00:00 [queued]>
[2025-03-06T06:18:10.846+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_horoscope_dag.send_email scheduled__2025-03-05T06:17:53.686903+00:00 [queued]>
[2025-03-06T06:18:10.846+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-03-06T06:18:10.850+0000] {taskinstance.py:1382} INFO - Executing <Task(EmailOperator): send_email> on 2025-03-05 06:17:53.686903+00:00
[2025-03-06T06:18:10.852+0000] {standard_task_runner.py:57} INFO - Started process 57 to run task
[2025-03-06T06:18:10.854+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_horoscope_dag', 'send_email', 'scheduled__2025-03-05T06:17:53.686903+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/email_dag.py', '--cfg-path', '/tmp/tmp2do88h3t']
[2025-03-06T06:18:10.854+0000] {standard_task_runner.py:85} INFO - Job 33: Subtask send_email
[2025-03-06T06:18:10.859+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-06T06:18:10.870+0000] {task_command.py:415} INFO - Running <TaskInstance: weather_horoscope_dag.send_email scheduled__2025-03-05T06:17:53.686903+00:00 [running]> on host a6553d1b92df
[2025-03-06T06:18:10.873+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-06T06:18:10.897+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='liuzifeng_r@163.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_horoscope_dag' AIRFLOW_CTX_TASK_ID='send_email' AIRFLOW_CTX_EXECUTION_DATE='2025-03-05T06:17:53.686903+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-05T06:17:53.686903+00:00'
[2025-03-06T06:18:10.903+0000] {base.py:73} INFO - Using connection ID 'smtp_default' for task execution.
[2025-03-06T06:18:10.904+0000] {email.py:270} INFO - Email alerting: attempt 1
[2025-03-06T06:18:13.228+0000] {email.py:282} INFO - Sent an alert email to ['liuzifeng_r@163.com']
[2025-03-06T06:18:14.076+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=weather_horoscope_dag, task_id=send_email, execution_date=20250305T061753, start_date=20250306T061810, end_date=20250306T061814
[2025-03-06T06:18:14.093+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-06T06:18:14.105+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
